bedrock/careers/management/commands/sync_greenhouse.py

```diff
@@ -0,0 +1,165 @@
+import html
+import re
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+import bleach
+import requests
+from html5lib.filters.base import Filter
+
+from bedrock.careers.models import Position
+
+GREENHOUSE_URL = "https://api.greenhouse.io/v1/boards/{}/jobs/?content=true"
+# to see the raw data for debugging use this command:
+# curl 'https://api.greenhouse.io/v1/boards/mozilla/jobs/?content=true' | \
+# jq -r .jobs[0].content | sed 's/&lt;/</g' | sed 's/&quot;/"/g' | sed 's/&gt;/>/g'
+
+# based on bleach.sanitizer.ALLOWED_TAGS
+ALLOWED_TAGS = [
+    "a",
+    "abbr",
+    "acronym",
+    "b",
+    "blockquote",
+    "button",
+    "code",
+    "div",
+    "em",
+    "h1",
+    "h2",
+    "h3",
+    "h4",
+    "h5",
+    "h6",
+    "i",
+    "img",
+    "li",
+    "ol",
+    "p",
+    "small",
+    "span",
+    "strike",
+    "strong",
+    "ul",
+]
+ALLOWED_ATTRS = [
+    "alt",
+    "class",
+    "href",
+    "id",
+    "src",
+    "srcset",
+    "style",
+    "rel",
+    "title",
+]
+ALLOWED_STYLES = [
+    "font-weight",
+]
+
+
+class HeaderConverterFilter(Filter):
+    def __iter__(self):
+        for token in Filter.__iter__(self):
+            if token["type"] in ["StartTag", "EndTag"]:
+                if token["name"] in ["h1", "h2", "h3"]:
+                    token["name"] = "h4"
+            yield token
+
+
+cleaner = bleach.sanitizer.Cleaner(tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRS, styles=ALLOWED_STYLES, strip=True, filters=[HeaderConverterFilter])
+
+
+class Command(BaseCommand):
```

Comment:
When we add this in, would be good to decorate with `@bedrock.utils.management.decorators.alert_sentry_on_exception`, especially if it's going to be called from `./bin/run-db-update.sh`